spinnaker/spinnaker_system/google_server_group_test.py

```diff
@@ -10,11 +10,25 @@
 # Spinnaker modules.
 import spinnaker_testing as sk
 import spinnaker_testing.gate as gate
-import google_quota_test as quota
 import citest.base
 
 
-class GoogleServerGroupTestScenario(quota.GoogleQuotaTestScenario):
+class GoogleServerGroupTestScenario(sk.SpinnakerTestScenario):
+
+  MINIMUM_PROJECT_QUOTA = {
+      'INSTANCE_TEMPLATES': 1,
```

Comment:
Since we reuse these, maybe a quota constants file would be helpful to have rather than poking around with gcloud to find the quota keys.